src/go/rpk/pkg/cli/cluster/storage/list-mount.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package storage
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/redpanda-data/common-go/rpadmin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newMountList(fs afero.Fs, p *config.Params) *cobra.Command {
+	var filter string
+	cmd := &cobra.Command{
+		Use:   "list-mount",
+		Short: "List mount/unmount operations",
+		Long: `List mount/unmount operations on a topic to the Redpanda Cluster from Tiered Storage.
```

Comment:
```suggestion
		Long: `List mount/unmount operations on a topic to the Cluster from Tiered Storage.
```
We wrap to 80 cols, we could to this to avoid a newline on the 'title' of the long text